Components/Sources/Components/Components/Editors/Common Views/Base/WKEditorToolbarButton.swift

```diff
@@ -52,49 +55,87 @@ class WKEditorToolbarButton: WKComponentView {
     }
     
     override func appEnvironmentDidChange() {
-        updateColors()
+        
+        guard let button else {
+            return
+        }
+        
+        let buttonConfig = createButtonConfig(image: image)
+        button.configuration = buttonConfig
+        button.configurationUpdateHandler = buttonConfigurationUpdateHandler(button:)
```

Comment:
Cool! I didn't know about this!

Reply:
@mazevedofs yep, it's the newer style of button configuration for UIKit. I can't remember if this was required for this particular button, but I needed it to add padding to the buttons in `WKEditorMultiButton` without triggering project warnings. We'll need to use a similar approach in `WKSmallMenuButton` to remove those warnings.